source/mir/combinatorics/package.d

```diff
@@ -251,6 +300,9 @@ Params:
 
 Returns:
     Forward range, which yields the permutations
+
+See_Also:
+    $(LREF Permutations), $(LREF dispose)
```

Comment:
this is not good enough  
> On 15 апр. 2016 г., at 18:08, Sebastian Wilzbach <notifications@github.com> wrote:
> 
> In source/mir/combinatorics/package.d <https://github.com/DlangScience/mir/pull/88#discussion_r59898839>:
> 
> > @@ -251,6 +300,9 @@ Params:
> >  
> >  Returns:
> >      Forward range, which yields the permutations
> > +
> > +See_Also:
> > +    $(LREF Permutations), $(LREF dispose)
> I just tested and if I click on dispose it will get to such a page - is this good enough or should we trust that the unittest does the explanation?
> 
>  <https://cloud.githubusercontent.com/assets/4370550/14567575/5b2cef92-033d-11e6-8fba-96ccd58c8996.png>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/DlangScience/mir/pull/88/files/21c97a5060d40c21e7e415ee78bab94c461e591f#r59898839>